src/GraphQLParser/Visitors/SDLPrinter.cs

```diff
@@ -721,7 +721,7 @@ protected override async ValueTask VisitArgumentAsync(GraphQLArgument argument,
     protected override async ValueTask VisitArgumentsDefinitionAsync(GraphQLArgumentsDefinition argumentsDefinition, TContext context)
     {
         await VisitAsync(argumentsDefinition.Comments, context).ConfigureAwait(false);
-        await context.WriteAsync("(").ConfigureAwait(false);
+        await VisitAsync(LiteralNode.Wrap("("), context).ConfigureAwait(false);
```

Comment:
General note - any literal printed first (after optional comment or description node) in `VisitXXX` method should be wrapped into `LiterlaNode` for proper indentation.